db/postgres.go

```diff
@@ -43,6 +43,7 @@ type PostgreSQL struct {
 	KeyFieldName          string
 	ValueFieldName        string
 	PartnersJSONFieldName string
+	OptExtraIndexes       map[string]string
```

Comment:
O modelo dos dados já diz q é opcional dado que um `map` pode estar vazio.
```suggestion
	ExtraIndexes       map[string]string
```